core/src/main/java/org/apache/iceberg/SystemConfigs.java

```diff
@@ -72,6 +72,19 @@ private SystemConfigs() {}
           8,
           Integer::parseUnsignedInt);
 
+  /** @deprecated will be removed in 2.0.0; use name mapping instead */
+  @Deprecated
+  public static final ConfigEntry<Boolean> NETFLIX_UNSAFE_PARQUET_ID_FALLBACK_ENABLED =
+      new ConfigEntry<>(
+          "iceberg.netflix.unsafe-parquet-id-fallback.enabled",
```

Comment:
I am not sure including Netflix in the property name is strictly required. I'd consider just calling it `iceberg.unsafe-parquet-id-fallback.enabled` or `iceberg.parquet.unsafe-id-fallback.enabled`. Up to you, though.

Reply:
I like having Netflix in the name because it signals that people should not use this. It is not a feature, it is an artifact of Iceberg's history.